zimbra/src/main/java/fr/openent/zimbra/service/impl/MessageService.java

```diff
@@ -301,6 +301,11 @@ private void transformMessageZimbraToFront(JsonObject msgZimbra, JsonObject msgF
             folder = FrontConstants.FOLDER_INBOX;
         }
 
+        if (msgZimbra.containsKey(MSG_EMAILID)) {
+            String mid = msgZimbra.getString(MSG_EMAILID).substring(1, msgZimbra.getString(MSG_EMAILID).length() - 1);
```

Comment:
We don't need the `<>` in Message-ID `<234483579.4178.1674212806701.JavaMail.zimbra@mail.com>`

Reply:
Tu peux vérifier que le size soit supérieur strictement à 1 pour éviter les erreurs